bc_obps/registration/schema/v2/multiple_operator.py

```diff
@@ -0,0 +1,82 @@
+from typing import Optional
+from registration.models.business_structure import BusinessStructure
+from registration.schema.v1.business_structure import validate_business_structure
+from registration.schema.validators import validate_cra_business_number
+from ninja import Field, ModelSchema
+from common.constants import AUDIT_FIELDS
+from registration.constants import BC_CORPORATE_REGISTRY_REGEX
+from registration.models import MultipleOperator
+from pydantic import field_validator
+
+
+class MultipleOperatorIn(ModelSchema):
+    """
+    Schema for the Multiple Operator part of the operation form
+    """
+
+    id: Optional[int] = None
+    legal_name: str = Field(..., alias="mo_legal_name")
+    trade_name: str = Field(..., alias="mo_trade_name")
+    business_structure: str = Field(..., alias="mo_business_structure")
+    cra_business_number: int = Field(..., alias="mo_cra_business_number")
+    bc_corporate_registry_number: Optional[str] = Field(
+        None, alias="mo_bc_corporate_registry_number", pattern=BC_CORPORATE_REGISTRY_REGEX
+    )
+    street_address: Optional[str] = Field(None, alias="mo_attorney_street_address")
+    municipality: Optional[str] = Field(None, alias="mo_municipality")
+    province: Optional[str] = Field(None, alias="mo_province")
+    postal_code: Optional[str] = Field(None, alias="mo_postal_code")
+
+    @field_validator("cra_business_number")
+    @classmethod
+    def validate_cra_business_number(cls, value: int) -> Optional[int]:
+        return validate_cra_business_number(value)
+
+    @field_validator("business_structure")
+    @classmethod
+    def validate_business_structure(cls, value: str) -> BusinessStructure:
+        return validate_business_structure(value)
+
+    class Meta:
+        model = MultipleOperator
+        fields = ["id"]
+        populate_by_name = True
+
+
+class MultipleOperatorOut(ModelSchema):
+    """
+    Schema for the MultipleOperator model
+    """
+
+    mo_legal_name: str = Field(..., alias="legal_name")
+    mo_trade_name: str = Field(..., alias="trade_name")
+    mo_cra_business_number: int = Field(..., alias="cra_business_number")
+    mo_bc_corporate_registry_number: str = Field(
+        ..., alias="bc_corporate_registry_number", pattern=BC_CORPORATE_REGISTRY_REGEX
+    )
+    mo_business_structure: str = Field(..., alias="business_structure")
+    mo_physical_street_address: str = Field(..., alias="physical_address.street_address")
+    mo_physical_municipality: str = Field(..., alias="physical_address.municipality")
+    mo_physical_province: str = Field(..., alias="physical_address.province")
+    mo_physical_postal_code: str = Field(..., alias="physical_address.postal_code")
```

Comment:
we don't need physical address for any operator anymore, just mailing address. All the physical address stuff can be removed

Reply:
We don't need at out schema at all at this point because we don't show any of the steps read-only, so I'll remove the whole thing.